packages/components/button/Button.mdx

```diff
@@ -62,6 +62,18 @@ Contentful buttons are available in 3 different sizes:
 </Stack>
 ```
 
+### Rendered as different Elements
+
+Contentful buttons can be rendered as a-tag which allows also use them with a href.
+
+```jsx
+<Stack>
```

Comment:
Looking good. Just one thing that I am not sure about: why do we need `<Stack>` in this example?

Reply:
haha, yeah i was just lazy copy&pasting from the other examples!